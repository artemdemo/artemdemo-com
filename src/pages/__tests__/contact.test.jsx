import React from 'react';
import renderer from 'react-test-renderer';
import Contact from '../contact';

jest.mock('../../components/Layout/Layout');
jest.mock('../../components/ContactForm/ContactForm');
jest.mock('../../components/Container/Container');

describe('Contact', () => {
    it('should render', () => {
        const tree = renderer.create(
            <Contact
                data={{
                    site: {
                        siteMetadata: {
                            title: 'Some title',
                            description: 'Some description',
                        },
                    },
                }}
                location={{}}
            />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});