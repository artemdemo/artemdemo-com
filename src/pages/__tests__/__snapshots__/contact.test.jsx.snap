// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should render 1`] = `
<div
  data-description="Some description"
  data-metaTitle="Some title"
  data-mock="Layout"
  data-title="Some title"
>
  <h1>
    Contact me
  </h1>
  <p>
    Hi there!
  </p>
  <p>
    If you want to drop me a line, you're more then welcome to do so. As you can see, I'm using form instead of simple email address. There is some reasons for that:
  </p>
  <ul>
    <li>
      I don't want get more spam that I'm already receiving.
    </li>
    <li>
      By using dedicated form I will have reserved copy of the email, in case I removed one from my inbox by accident.
    </li>
    <li>
      And finally I think it's simply faster, to the user. You don't need to click on email, wait until your client will open and then do exactly what you can do on this very page :)
    </li>
  </ul>
  <div
    className="contact-form-loader"
  >
    Form is loading...
  </div>
  <iframe
    className="contact-form"
    frameBorder="0"
    height="822"
    marginHeight="0"
    marginWidth="0"
    onLoad={[Function]}
    src="https://docs.google.com/forms/d/e/1FAIpQLSfbcq5MKljpEZg27bOOj5OqdBP4ARBdikUs4G2zuqIgBYE4KA/viewform?embedded=true"
    width="640"
  >
    Loading...
  </iframe>
</div>
`;
